Musigwa/SendIT-API
# [Feature #163356948] Create the new parcel delivery order

## What does this PR do (?)
It the logged in user to create a new parcel delivery order.
## Description of Task to be completed (?)
The user should be able to create the new parcel order from his/her account.
If any logged in user wants to create a parcel delivery order, there is a button to add it.
When he/she fills out the form correctly and submit, the order must be created.
## How should this be manually tested (?)
After cloning this repository and head to this branch,

- Remember to tweak your database credentials via config directory.
- Start your web pages using any browser as a debugger.
- Create an account if you haven't yet and try logging out to see or,
- Log into the application using the right credentials if you already have an account,
- Locate and click the bottom-right (plus) button,
- Fill in the new order form correctly and submit it. 

## Any background context you want to provide (?)
I'm using ES6 features, bear with it and run the build command yarn build if needed and change the package.json to match every command setup.

## What are the relevant pivotal tracker stories (?)
- 163356948
